sys-apps/qdirstat/qdirstat-1.6.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit qmake-utils xdg-utils
+
+DESCRIPTION="Qt-based directory statistics"
+HOMEPAGE="https://github.com/shundhammer/qdirstat"
+SRC_URI="https://github.com/shundhammer/qdirstat/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="
+	dev-qt/qtgui:5
+	dev-qt/qtcore:5
+	dev-qt/qtwidgets:5
+	sys-libs/zlib
+"
+
+RDEPEND="
+	${DEPEND}
+	dev-lang/perl
+	dev-perl/URI
+"
+
+src_prepare() {
+	default
+
+	# Fix QA warning about incorrect use of path
```

Comment:
```
 * Messages for package sys-apps/qdirstat-1.6:

 * The ebuild is installing to one or more unexpected paths:
 * 
 *   /usr/share/doc/qdirstat
 * 
 * Please fix the ebuild to use correct FHS/Gentoo policy paths.
```
:(

Reply:
Sorry, commited older eBuild. This one is now correct and qa error is fixed :-)